R/conda.R

```diff
@@ -758,17 +758,21 @@ conda_run <- function(cmd, args = c(), conda = "auto", envname = NULL,
                       run_args = c("--no-capture-output"), ...) {
 
   conda <- conda_binary(conda)
-  envname <- condaenv_resolve(envname)
+  if(!identical(envname, "base"))
+    envname <- condaenv_resolve(envname)
```

Comment:
I wonder if this check should be part of `condaenv_resolve()` itself? (So that `condaenv_resolve("base")` always returns "base")